src/content/developers/docs/consensus-mechanisms/index.md

```diff
@@ -68,6 +68,17 @@ A proof-of-stake system is kept secure by the fact that you'd need 51% of the to
 
 More on [proof-of-stake (PoS)](/developers/docs/consensus-mechanisms/pos/)
 
+### Sybil Resistance & Chain Selection
+
+Now technically, both Proof of Work (PoW) and Proof of Stake (PoS) are not consensus protocols by themselves, but they are often referred to as such for simplicity. PoW and PoS are actually Sybil Resistance mechanisms and block author selectors, they are a way to decide who is the author of the latest block. It's this mechanism combined with at least a Chain selection rule that makes up a true consensus mechanism.
+
+Sybil Resistance is a measure of how a protocol fairs against a [sybil attack](https://en.wikipedia.org/wiki/Sybil_attack). Sybil attacks are when one user or group pretends to be many users. This is essential in blockchain so that the blockchain is executed in a decentralized manner, and miners and validators are rewarded equally. PoW and PoS protect against this by making users expend a lot of engergy or put up a lot of collateral, making it really hard for one 
```

Comment:
Similarly in 1st sentence, "Sybil resistance" and "Sybil attack"

> Sybil attacks are when one user or group pretends to be many users. This is essential in blockchain 

Subject switched... makes it sound like the attacks are essential.

> This is essential in blockchain so that the blockchain is executed in a decentralized manner, and miners and validators are rewarded equally.

This reads a little funny with "blockchain" twice... maybe something like "Resistance to this type of attack is essential for a decentralized blockchain, and enables miners and validators to be rewarded equally based on resources put in."

> ... of collateral, making it really hard for one 

Hah, lose your train of thought? 😉

Reply:
Fixed... and yes I did

Reply:
lol